kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -373,6 +373,7 @@ def exec_foundry_prove(
     kore_rpc_command: str | Iterable[str] = ('kore-rpc',),
     smt_timeout: int | None = None,
     smt_retry_limit: int | None = None,
+    failure_info: bool = False,
```

Comment:
We should probably have this default match the CLI argument default, which is `True`.

Reply:
Updated.